### ISSUE TITLE
Status change email giving wonky disk space left notifications

### ISSUE THREAD

**Issue by zoof** [mail-in-a-box/mailinabox#592]
Yesterday:

System -- Previously:
=====================
✓  The disk has less than 10% space left.

System -- Currently:
====================
?  The disk has less than 10% space left.

today:

System -- Previously:
=====================
?  The disk has less than 10% space left.

System -- Currently:
====================
✓  The disk has less than 10% space left.

while the system has had roughly 30% free for a couple of weeks now.

**Comment by sbidin** [comment]
I have the same issue.

It started occurring once I added more disk space to the system (upgraded a DigitalOcean droplet). Here's a sample email:

    System -- Previously:
    ?  The disk has less than 10% space left.

    System -- Currently:
    ✓  The disk has less than 10% space left.

Could mailinabox be caching its notion of total disk size from when it was first installed?

**Comment by aspdye** [comment]
Can you give us more details about the instance and the specific setup :grin: THX

**Comment by sbidin** [comment]
Sure. This instance is a boring DigitalOcean droplet. It used to have a 30GB disk. It was then powered off and the disk was resized to 40GB. During this process the newest (current stable) version of mailinabox was being used. I'd be glad to provide any other details that you consider relevant.

**Comment by zoof** [comment]
I run it on a 24GB Linode.  This strange message happens rarely but it has happened (not for a while though).  I've tweaked a couple of things (installed pflogsum and added /etc/fail2ban/jail.d files to ban permanently and increase findtimes) but nothing that should affect remaining estimated storage.  I don't suppose the nightly script generates any logs that would help diagnose this.

**Comment by sbidin** [comment]
The messages have not been occurring for a week now, even though nothing else seems to have changed.

**Comment by sbidin** [comment]
It happened again.

```
System -- Previously:
=====================
✓  The disk has less than 10% space left.

System -- Currently:
====================
?  The disk has less than 10% space left.
```

This is the output of `df -h`:
```
~ λ df -h
Filesystem                 Size  Used Avail Use% Mounted on
udev                       991M  8.0K  991M   1% /dev
tmpfs                      201M  428K  200M   1% /run
/dev/disk/by-label/DOROOT   40G   26G   12G  70% /
none                       4.0K     0  4.0K   0% /sys/fs/cgroup
none                       5.0M     0  5.0M   0% /run/lock
none                      1001M   16K 1001M   1% /run/shm
none                       100M     0  100M   0% /run/user
```

**Comment by JoshData** [comment]
Fixed by #932.
